packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.js

```diff
@@ -259,4 +259,38 @@ export default class QuanticResultQuickview extends LightningElement {
       focusTarget.focus();
     }
   }
+
+  getLsatFocusableElement(node) {
+    if (!node || node.nodeType === 3) return null;
+
+    const lastFocusable = Array.from(node.childNodes)
+      .map((item) => this.getLsatFocusableElement(item))
+      .filter((item) => !!item);
+
+    if (lastFocusable.length > 0) {
+      return lastFocusable[lastFocusable.length - 1];
+    } else if (isFocusable(node)) {
+      return node;
+    }
+    return null;
```

Comment:
Put this in a method exposed in Utils

```suggestion
    const quickviewContainer = this.template.querySelector('.quickview__content-container');
    if(quickviewContainer) {
        return getLastFocusableElement(quickviewContainer);
    }
```